code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1630,3 +1630,14 @@
 	description = "blue sparkles that get everywhere"
 	color = "#4040FF" //A blueish color
 	glitter_type = /obj/effect/decal/cleanable/glitter/blue
+
+/datum/reagent/pukonium
```

Comment:
AAAAAAAAAAH TACO WHY. I AM REPLYING TO PEOPLE WHO WANT TO TRANSFER IT TO A BASE "BAD FOOD" REAGNENT. I AM LITERALLY DEFENDING YOUR REAGNENT.